src/elm/Main.elm

```diff
@@ -2903,34 +2942,36 @@ loadBuildPage model org repo buildNumber lineFocus =
         sameBuild =
             isSameBuild ( org, repo, buildNumber ) model.page
 
-        pageSet =
-            { model | page = Pages.Build org repo buildNumber lineFocus }
+        updateBuild =
+            if not sameBuild then
+                setBuild org repo buildNumber
+
+            else
+                identity
```

Comment:
this, and the other changes made below this are mostly cleanup and adopting the style merged in recently